ethers-contract/ethers-contract-abigen/Cargo.toml

```diff
@@ -15,23 +15,30 @@ keywords = ["ethereum", "web3", "celo", "ethers"]
 
 [dependencies]
 ethers-core = { version = "^1.0.0", path = "../../ethers-core", features = ["macros"] }
+ethers-etherscan = { path = "../../ethers-etherscan", default-features = false, optional = true }
```

Comment:
why do we need etherscan here?

Reply:
we have duplicate code for parsing the get request when we could just use the specialized crate in the same project